src/main/java/duke/MainWindow.java

```diff
@@ -0,0 +1,65 @@
+package duke;
+
+import duke.ui.Ui;
```

Comment:
There should be a newline between "import.duke.ui.Ui;" and "import javafx.fxml.FXML;".